schemas/pages/home-page.ts

```diff
@@ -9,7 +9,7 @@ const textFields = [
   defineField({
     title: "Descripción",
     name: "description",
-    type: "string",
+    type: "block",
```

Comment:
@alexxuss1996 I think I didn't explain this very well!

Here (line 12) it should stay as `string`, and then create a new description field with the type `block` only for both of the pointed fields

Reply:
Oh, ok, will be made!